clarity/src/vm/clarity.rs

```diff
@@ -305,7 +317,7 @@ pub trait TransactionConnection: ClarityConnection {
         max_execution_time: Option<std::time::Duration>,
     ) -> Result<(Value, AssetMap, Vec<StacksTransactionEvent>), Error>
     where
-        F: FnOnce(&AssetMap, &mut ClarityDatabase) -> bool,
+        F: FnOnce(&AssetMap, &mut ClarityDatabase) -> Option<String>,
```

Comment:
Ah, I see you modified `abort_call_back` here. Can you update the doc comment for this function?

Reply:
Oops, yes, I'll update the comment. Thanks for catching that.

Reply:
Ok, I updated the comments here and other places about this call back in 2d218722005a99994b4442237fc4e8d720c475ff.